### PR TITLE
Chunyueleung onemean

### PR DESCRIPTION
**Pull request by aryue123321**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
 <div id="main-menu">
   <ul>
     <li><a href="/oneProportion/" class="btn">One Proportion</a></li>
+    <li><a href="/oneMean/" class="btn">One Means</a></li>
```
Comment (MatthewMerrill) [inline src/index.html]:
```suggestion
    <li><a href="/oneMean/" class="btn">One Mean</a></li>
```

File: src/oneMean/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      </select>
+    </div>
+    <div class="view">
+      <div class="chart-wrap"><canvas id="data-chart-1"></canvas></div>
+    </div>
+  </div>
+  <h2>Step Two: Run Simulations</h2>
+  <div id="one-mean-sample">
+    <div class="control">
+      <a onclick="twoMean.runSim()" onclick="" class="btn">Run Simulation</a>
```
Comment (MatthewMerrill) [inline src/oneMean/index.html]: nitpick: Can you change this to a button?

File: src/oneMean/oneMean.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      return (acc += `<option value="${x}">${x}</option>`);
+    }, '<option value="null">Please Select Variable</option>');
+
+    dropDownElement.innerHTML = html;
+  }
+
+  updateData(varName, data) {
+    this.dataChart1.updateLabelName(0, varName);
+    this.dataChart1.setDataFromRaw([data[varName]]);
+    this.dataChart1.changeDotAppearance(1, undefined);
```
Comment (MatthewMerrill) [inline src/oneMean/oneMean.js]: Does this make the points one pixel wide?

Comment (aryue123321) [reply src/oneMean/oneMean.js]: Yes, because  some one mean dataset are so big, some over 10000 pts

File: src/oneMean/index.html (hunk 2 above)
Comment (aryue123321) [reply src/oneMean/index.html]: ty

File: src/index.html (hunk 1 above)
Comment (aryue123321) [reply src/index.html]: ty

**Comment by aryue123321** [comment]
image

**Pull request merged**